docs/developer-docs/integrations/sns/tokenomics/sns-checklist.md

```diff
@@ -57,46 +58,26 @@ It may not be possible to fully decentralize the dapp at the time of the SNS lau
 If the dapp does have dependencies of some kind that cannot be managed by the DAO, then those dependencies should be documented so the community is aware of potential risks. When possible, there should also be a plan for how to bring these services on-chain and under control of the DAO.
 
 ## 1.4. Create SNS configuration file
-The SNS is initialized using a configuration file, and the parameters in this file will set up token name, token supply, token distribution, transaction fees, dissolve delays and more. The configuration will be a part of the decentralization proposal, and is approved by the NNS DAO.
-
-For initializing an SNS for your dapp, the configuration parameters must use the documentation created with the SNS tokenomics tool (section 1.1.5 above) to create the YAML configuration file. The [OpenChat config file](https://github.com/open-ic/open-chat/blob/master/sns/config/sns.yml) is a good example of how to format the file (note: the latest version of SNS may have new/different parameters). Add comments, explaining the parameters and token allocation, in the configuration file for transparency and ease of reading.
+The SNS is initialized using an NNS proposal that defines the initial parameters, thus the initial configuration is approved by the NNS DAO.
+The proposal can be created from a YAML configuration file and defines things like the token name, token supply, token distribution, transaction fees, dissolve delays and more. 
 
-Geo-restriction can be used to exclude users with IPs in specific countries with the “restricted_countries” parameter, which takes an array of country codes like this: `restricted_countries: [ "CH" ]` 
+For initializing an SNS for your dapp, the configuration parameters must use the documentation created with the SNS tokenomics tool (section 1.1.5 above) to create the YAML configuration file. The [OpenChat config file](https://github.com/open-ic/open-chat/blob/master/sns/config/sns.yml) is a good example of how to format the file (note: the latest version of SNS may have new/different parameters). Add comments, explaining the parameters and token allocation, in the configuration file for transparency and ease of reading. 
```

Comment:
Like in the other comment, lets link the new template file here, or to the `Sns Initial parameters` doc section. The OC is the old config method so I would at the very least not link it, or emphasize that this is the old configuration file method

Reply:
I added the remark that OC is and old format.

For the other part, would you mind making a suggestion? I don't know if we say enough (yet) in the init parameter part of the docs to clarify all quesitons, but maybe we would afte resolving the conversation above

Reply:
Added links to yaml template and example and linked to preparation side.